src/static/riot/queues/list.tag

```diff
@@ -0,0 +1,282 @@
+<queues-list>
+    <div class="ui icon input">
+        <input type="text" placeholder="Search by name..." ref="search" onkeyup="{filter.bind(this, undefined)}">
+        <i class="search icon"></i>
+    </div>
+    <div class="ui checkbox" onclick="{ filter.bind(this, undefined) }">
+        <label>Show Public Queues</label>
+        <input type="checkbox" ref="public">
+    </div>
+    <div class="ui green right floated labeled icon button" onclick="{ show_modal.bind(this, undefined) }"><i class="add circle icon"></i>
+        Create Queue
+    </div>
+    <table class="ui {selectable: queues.length > 0} celled compact table">
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th width="150px">Owner</th>
+            <th width="125px">Created</th>
+            <th width="50px">Public</th>
+            <th class="right aligned" width="150px">Actions</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr each="{ queue in queues }" class="queue-row">
+            <td>{ queue.name }</td>
+            <td>{ queue.owner }</td>
+            <td>{ timeSince(Date.parse(queue.created_when)) } ago</td>
+            <td class="center aligned">
+                <i class="checkmark box icon green" if="{ queue.is_public }"></i>
+            </td>
+            <td class="right aligned">
+                <span class="popup" data-tooltip="{queue.broker_url}" data-position="left center">
+                    <i class="grey icon eye popup-button" if="{ !!queue.broker_url }"></i>
+                </span>
+                <span class="popup" data-tooltip="Copy Broker URL">
+                    <i class="icon copy outline popup-button" if="{ !!queue.broker_url }"
+                       onclick="{ copy_queue_url.bind(this, queue) }"></i>
+                </span>
+                <span class="popup" data-tooltip="Edit Queue">
+                    <i class="blue icon edit popup-button" if="{ queue.is_owner && !!queue.broker_url }" onclick="{ show_modal.bind(this, queue) }"></i>
+                </span>
+                <span class="popup" data-tooltip="Delete Queue">
+                    <i class="red icon trash alternate outline popup-button" if="{ queue.is_owner && !!queue.broker_url }"
+                       onclick="{ delete_queue.bind(this, queue) }"></i>
+                </span>
+            </td>
+        </tr>
+
+        <tr if="{queues.length === 0}">
+            <td class="center aligned" colspan="4">
+                <em>No Queues Yet!</em>
+            </td>
+        </tr>
+        </tbody>
+        <tfoot>
+        <!-------------------------------------
+                  Pagination
+        ------------------------------------->
+        <tr if="{queues.length > 0 && _.get(pagination, 'next')}">
+            <th colspan="6">
```

Comment:
`colspan=5` ?

Reply:
I did verify pagination and fixed a small issue with it. It should be working as intended.